packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -615,12 +627,16 @@ class WSIViewport extends Viewport {
   protected canvasToIndex = (canvasPos: Point2): Point2 => {
     const transform = this.getTransform();
     transform.invert();
-    return transform.transformPoint(canvasPos);
+    return transform.transformPoint(
```

Comment:
Can you use Math.abs on the result here - I think that will get us close to the right value, or on initial setup, figure out which attribute is negative and store a negative multiplier for it.